### PR TITLE
Fix Azure Data Explorer Operator

### PR DESCRIPTION
**Pull request by kentdanas**
Updated adx operator to return a json serializable result.
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by kentdanas** [comment]
@kaxil PR with our changes submitted here. Let me know if there's anything else needed from my side!

**Review by kaxil — APPROVED** [review]
Thanks @kentdanas :)

Let's wait for the CI results, once it passes I will happily merge it.

**Pull request merged**
